NanoVNASaver/NanoVNASaver.py

```diff
@@ -905,6 +905,7 @@ def loadSweepFile(self):
             self.data21 = []
             t = Touchstone(filename)
             t.load()
+            # shouldn't modify read file. even destroys it if read fails 
```

Comment:
It doesn't, so this comment should be removed. NanoVNASaver.saveData saves the data to the application, and does not touch the files.